### PR TITLE
[Operational Management] Add account-resource command to the operational tool.

### PR DESCRIPTION
**Pull request by JoshLind**
## Motivation

This PR adds the account-resource command to the operational tool. This is useful to retrieve and print the account resource of an account address on-chain (e.g., to verify the new authentication key after an authentication key has been rotated by an operator).

This PR achieves this using 2 commits:
1. Adding the account-resource command to the tool.
2. Updating the rotate operator key smoke tests to use the operational tool to verify the change.

Note: we might have to look at building our own pretty print tool to better display the output. For example, serde_json::to_string_pretty() seems to print out vectors by printing each item on its own line (see below). This might be useful for some cases, but it's kind of ugly for bytes... Anyway, for now, the command prints out all information relevant to an AccountResource struct.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

## Test Plan

All tests pass locally. Moreover, building the tool and pointing it a JSON RPC endpoint produces this output:

{
  "Result": {
    "authentication_key": [
      30,
      20,
      196,
      62,
      93,
      75,
      5,
      89,
      147,
      207,
      116,
      74,
      245,
      72,
      199,
      114,
      97,
      47,
      218,
      189,
      11,
      151,
      60,
      73,
      57,
      218,
      147,
      99,
      26,
      110,
      169,
      131
    ],
    "withdrawal_capability": {
      "account_address": "612fdabd0b973c4939da93631a6ea983"
    },
    "key_rotation_capability": {
      "account_address": "612fdabd0b973c4939da93631a6ea983"
    },
    "received_events": {
      "count": 0,
      "key": "0000000000000000612fdabd0b973c4939da93631a6ea983"
    },
    "sent_events": {
      "count": 0,
      "key": "0100000000000000612fdabd0b973c4939da93631a6ea983"
    },
    "sequence_number": 0
  }
}

## Related PRs

None, but this is an item on the issue: https://github.com/libra/libra/issues/5082

### CODE REVIEW

**Comment by davidiw** [comment]
Can we adjust authentication_key to print out the hex formatted string instead?

File: testsuite/tests/libratest/smoke_test.rs [hunk 1]
```diff
@@ -1643,10 +1646,24 @@ fn test_operator_key_rotation_recovery() {
...
     assert_eq!(VMStatusView::Executed, vm_status);
 
+    // Verify that the operator key was updated on-chain
+    let mut storage: Storage = (&backend).try_into().unwrap();
+    let operator_account_string = storage
+        .get(OPERATOR_ACCOUNT)
+        .unwrap()
+        .value
+        .string()
+        .unwrap();
```
Comment (gregnazario) [inline testsuite/tests/libratest/smoke_test.rs:1656]: It's times like this, that Rust did what Kotlin did for throwable unwraps `!!` cause it's shorter :P

File: config/management/operational/src/test_helper.rs [hunk 2]
```diff
@@ -28,6 +30,25 @@ impl OperationalTool {
...
+    pub fn account_resource(
+        &self,
+        account_address: AccountAddress,
+    ) -> Result<AccountResource, Error> {
+        let args = format!(
+            "
+                {command}
+                --json-server {json_server}
+                --account-address {account_address}
+        ",
```
Comment (gregnazario) [inline config/management/operational/src/test_helper.rs:42]: Nit: The closing quote somehow is off tab

**Comment by JoshLind** [comment]
Thanks David and Greg! I'll follow this PR with another that cleans up the pretty_print output of the tool for this command (and does some general refactors!)

**Pull request closed without merging**